include/uv.h

```diff
@@ -1294,6 +1300,10 @@ struct uv_fs_s {
   UV_FS_PRIVATE_FIELDS
 };
 
+#undef UV_REQ_PRIVATE_FIELDS
+#define UV_REQ_PRIVATE_FIELDS UV_REQ_PRIVATE_FIELDS_TEMP
+#undef UV_REQ_PRIVATE_FIELDS_TEMP /* for uv_fs_s */
```

Comment:
> I'd add a union to `UV_REQ_FIELDS`

You mean I can change the size of `uv_fs_s`? (I thought all public struct sizes were frozen.)

Or do you mean change it to

```c
void* reserved[4]; // 4 instead of 5
union { /* ... */ } priv;
```

Reply:
Yes, like that but it's currently a 6 element array, hence why I wrote `void* reserved[5];` :-)

Reply:
Woops, can't read :-p thanks.